models/meshmodel/registry/host.go

```diff
@@ -0,0 +1,97 @@
+package registry
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/layer5io/meshkit/database"
+	"github.com/layer5io/meshkit/models/oam/core/v1alpha1"
+	"github.com/layer5io/meshkit/utils/kubernetes"
+	"gorm.io/gorm"
+)
+
+type Host struct {
```

Comment:
Are you moving this table out of memory and into the database? If so, use the `connections` table, not a new `hosts` table.